### PR TITLE
Create get counter by id api for counter page- getcounterHistoryById request

### PR DESCRIPTION
**Pull request by KoralMazor**

### CODE REVIEW

File: chillers-BE-API/src/routes/api/counters.js [hunk 1]
```diff
@@ -1,20 +1,104 @@
...
+    res.status(200).json({ counters });
+}
+catch (err) {
+    logger.error(`getAllCountersSettings failed: ${err.message}`);
+    res.status(400).json({ code: err.code, message: err.message });
+}
+
+};
+
+const getCounterHistoryById = async (req, res) => {
```
Comment (smartcity-hit) [inline chillers-BE-API/src/routes/api/counters.js:33]: Change function name to "getCounterById(counterId)"

File: chillers-BE-API/src/utils/countersService.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+const mongoose = require('mongoose');
+const { devicesSchema, countersSchema } = require('./schemas');
+
+
+/**
+ * * All the exported functions, All  of the methods & statics related to schemas
+ */
+
+const getCountersSettings= async () => {
```
Comment (smartcity-hit) [inline chillers-BE-API/src/utils/countersService.js:9]: instead this function, write the following function:
getDevices(deviceId){
....
const device = mongoose.model('Devices', devicesSchema, 'devices');
const collections = await device.find( { deviceType: deviceId } );
...
}

**Review by smartcity-hit — COMMENTED** [review]
pay attention for variables naming convention in javascript.

File: chillers-BE-API/src/routes/api/counters.js (hunk 1 above)
Comment (KoralMazor) [reply chillers-BE-API/src/routes/api/counters.js:33]: this function is return history data for range of dates for counters
so, I'm not changing that for now

File: chillers-BE-API/src/utils/countersService.js (hunk 2 above)
Comment (KoralMazor) [reply chillers-BE-API/src/utils/countersService.js]: for now I changes only the function name

**Comment by KoralMazor** [comment]
@MoriaCohavi  about your comment: pay attention for variables naming convention in javascript.
please comment in the place you see it and I'll double check the code again.

File: chillers-BE-API/src/routes/api/counters.js (hunk 1 above)
Comment (KoralMazor) [reply chillers-BE-API/src/routes/api/counters.js]: Done 
@MoriaCohavi 
thanks

**Pull request merged**